docs/sdk/serverless-signing/quick-start.md

```diff
@@ -7,557 +7,188 @@ import TabItem from '@theme/TabItem';
 
-```jsx
-import { LitNetwork } from "@lit-protocol/constants";
+<details>
+<summary>Click here to see how this is done</summary>
+<p>
 
-app.locals.litNodeClient = new LitJsSdk.LitNodeClientNodeJs({
-  alertWhenUnauthorized: false,
-  litNetwork: LitNetwork.Datil,
-});
-await app.locals.litNodeClient.connect();
-```
-
-The litNodeClient listens to network state, and those listeners will keep your Node.js process running until you explicitly disconnect from the Lit network. To stop the litNodeClient listeners and allow node to exit gracefully, call: 
-
-```jsx
-await app.locals.litNodeClient.disconnect();
-```
-
-## Install the Required Packages
-
-```jsx
-yarn add @lit-protocol/contracts-sdk
-yarn add @lit-protocol/lit-auth-client
-yarn add @lit-protocol/auth-helpers
-```
-
-### Set up a controller wallet
-
-To initialize a LitContracts client you need an Ethereum Signer. This can be a standard Ethereum wallet (ethers) or a PKP (more info on the latter [here](../../user-wallets/pkps/advanced-topics/auth-methods/add-remove-auth-methods)). Here, we're going to use a standard Ethereum wallet.
-
-:::warning
-You'll need to use ethers.js v5 with the Lit SDK. The Lit SDK is not compatible with ethers.js v6 or higher.
-:::
-
-### Initialize the `contracts-sdk`
-
-```jsx
-import { LitContracts } from '@lit-protocol/contracts-sdk';
-import { LitNetwork } from "@lit-protocol/constants";
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork, LIT_RPC } from "@lit-protocol/constants";
+import * as ethers from "ethers";
 
-const contractClient = new LitContracts({
-  signer: wallet,
-  network: LitNetwork.Datil,
+const litNodeClient = new LitNodeClient({
+  litNetwork: LitNetwork.DatilDev,
+  debug: false
 });
+await litNodeClient.connect();
 
-await contractClient.connect();
+const ethersWallet = new ethers.Wallet(
+  process.env.ETHEREUM_PRIVATE_KEY, // Replace with your private key
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
 ```
 
-:::note
-
-You’ll need to ensure you have some test tokens to pay for gas fees. You can claim test tokens from our verified [faucet](https://chronicle-yellowstone-faucet.getlit.dev/).
-
-:::
+</p>
+</details>
 
-## Authenticate with the Lit Network
-In order to interact with the nodes in the Lit Network, you will need to generate and present signatures. You can do this by generating a 'Session Sig'. Any signature compliant with EIP-4361 (also known as Sign in with Ethereum (SIWE)) cam be used for this. 
+### Generating Session Signatures
 
-### Obtaining a `SessionSigs` in the browser
+Session signatures are vital to authentication on the Lit network. They also are required for giving your session the ability to execute Lit Actions. Please note that there is more than one way to generate session signatures, and that those methods can be found on their dedicated pages in the [Session Signatures](../authentication/session-sigs/intro) section.
```

Comment:
> Session signatures are vital to authentication on the Lit network. They also are required for giving your session the ability to execute Lit Actions. Please note that there is more than one way to generate session signatures, and that those methods can be found on their dedicated pages in the [Session Signatures](../authentication/session-sigs/intro) section.

Consider "Session signatures are used to authenticate with and maintain an active connection to the nodes in the Lit network. They are required when executing a Lit Action or any other functionality (i.e. signing) with Lit. There is more than one way to generate session signatures, and that those methods can be found on their dedicated pages in the [Session Signatures](../authentication/session-sigs/intro) section."